app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,7 @@
 <resources>
     <string name="app_name">Campus Guide</string>
     <string name="title_activity_maps">Map</string>
+    <string name="placeholder">temp</string>
```

Comment:
In what situation would the placeholder be used?

Reply:
oh oops that was a leftover from when i was still trying to get the feature to work with a floating action button. i'll get rid of it o7